src/library/Text/Text.js

```diff
@@ -0,0 +1,165 @@
+/* @flow */
+import React, { Component } from 'react';
+import { createStyledComponent } from '../styles';
+
+type Props = {
+  /** Available horizontal alignments */
+  align?: 'start' | 'end' | 'center' | 'justify',
+  /** Available styles */
+  appearance?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'mouse' | 'prose',
```

Comment:
BTW, I really like the `element/appearance` props API.
`<Text element="h1" appearance="h3">...` is super clear.

Reply:
You can thank Victoria for that.

Reply:
Thanks @VictoriaVasys 🙌